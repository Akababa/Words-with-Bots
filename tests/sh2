ra +glnb[ap
lm -n=25
pr b
