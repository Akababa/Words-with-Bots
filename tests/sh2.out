rack: ADOTES[
   ROADSTER @11, 2 acr for 59 (hscore:59) (blank:1)
   LODESTAR @11, 2 acr for 59 (hscore:59) (blank:1)
   DELATORS @11, 3 acr for 59 (hscore:59) (blank:3)
   ADOPTERS @11, 3 acr for 59 (hscore:59) (blank:4)
    TORSADE @ 3, 4 dwn for 57 (hscore:57) (blank:3)
    TOASTED @ 2, 4 dwn for 57 (hscore:57) (blank:1)
    TOASTED @ 2, 4 dwn for 57 (hscore:57) (blank:5)
    ROASTED @ 2, 4 dwn for 57 (hscore:57) (blank:1)
    IODATES @ 2, 4 dwn for 57 (hscore:57) (blank:1)
    DONATES @ 4, 4 dwn for 57 (hscore:57) (blank:3)
    COASTED @ 2, 4 dwn for 57 (hscore:57) (blank:1)
    BOASTED @ 2, 4 dwn for 57 (hscore:57) (blank:1)
    TOADIES @ 2, 4 dwn for 55 (hscore:55) (blank:5)
    TOADIES @ 5, 4 dwn for 55 (hscore:55) (blank:5)
    SOLATED @ 4, 4 dwn for 55 (hscore:55) (blank:3)
    SOLATED @ 2, 4 dwn for 55 (hscore:55) (blank:3)
    DOTAGES @ 2, 4 dwn for 55 (hscore:55) (blank:5)
    DOTAGES @ 4, 4 dwn for 55 (hscore:55) (blank:5)
    DOTAGES @ 3, 4 dwn for 54 (hscore:54) (blank:5)
    TORSADE @ 1, 4 dwn for 53 (hscore:53) (blank:3)
    TOASTED @ 5, 4 dwn for 53 (hscore:53) (blank:5)
    TOASTED @ 5, 4 dwn for 53 (hscore:53) (blank:1)
    ROASTED @ 5, 4 dwn for 53 (hscore:53) (blank:1)
    PODESTA @ 4, 4 dwn for 53 (hscore:53) (blank:1)
    PODESTA @ 1, 4 dwn for 53 (hscore:53) (blank:1)
    IODATES @ 4, 4 dwn for 53 (hscore:53) (blank:1)
    DONATES @ 2, 4 dwn for 53 (hscore:53) (blank:3)
   DIASTOLE @ 1, 5 dwn for 53 (hscore:53) (blank:2)
    COASTED @ 5, 4 dwn for 53 (hscore:53) (blank:1)
    BOASTED @ 5, 4 dwn for 53 (hscore:53) (blank:1)
      STEAD @10, 6 acr for 52 (hscore:52)
      STEAD @10, 6 acr for 48 (hscore:48) (blank:3)
      STEAD @10, 6 acr for 48 (hscore:48) (blank:2)
      SALAD @10, 6 acr for 48 (hscore:48) (blank:3)
      AHEAD @10, 6 acr for 48 (hscore:48) (blank:2)
   PASTORED @11, 4 acr for 47 (hscore:47) (blank:1)
     VADOSE @10, 2 acr for 46 (hscore:46) (blank:1)
     TSADES @10, 1 acr for 46 (hscore:46) (blank:2)
     TREADS @10, 1 acr for 46 (hscore:46) (blank:2)
     TRADES @10, 1 acr for 46 (hscore:46) (blank:2)
board:
012345678901
1*.!.....!.*
2.?...?...?.
3!.*.'.'.*.!
4...*...*...
5..'.....B..
6.?...P..A?.
7..'.LANDS..
8...*.N.*I..
9!.*.'G'.LOX
0.?...?..A?.
1*.!.....R.*
